src/main/java/org/kainos/ea/controller/JobSpecController.java

```diff
@@ -0,0 +1,23 @@
+package org.kainos.ea.controller;
+
+import io.swagger.annotations.Api;
+import javax.ws.rs.Path;
+import org.kainos.ea.service.JobSpecService;
+import org.kainos.ea.utility.DatabaseConnector;
+import org.kainos.ea.validator.JobSpecValidator;
+
+@Api("Kainos Job Application")
+@Path("/job-specification")
+public class JobSpecController {
+
+  private static JobSpecService jobSpecService;
+  private static JobSpecValidator jobSpecValidator;
+
+  public JobSpecController() {
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+//    jobSpecService = new JobSpecService(new JobSpecDao(), databaseConnector);
+    jobSpecValidator = new JobSpecValidator();
```

Comment:
Is there an issue with leaving this empty as we have with the other files?

Reply:
The code for these other classes will be written this morning, I've not fleshed it out and it shows errors until these are made

Reply:
Can I address this in the next user story I work on?

Reply:
I was thinking that we might be able to leave all these lines out to let them be added in the story they're needed for, was it that that was causing problems?

Reply:
Sounds good! Yes, these were causing problems so I commented a line out. I'll delete them and all should be well now.